SDKAndroid/src/main/java/app/cybrid/sdkandroid/util/Pooling.kt

```diff
@@ -0,0 +1,26 @@
+package app.cybrid.sdkandroid.util
+
+import android.os.Handler
+import android.os.Looper
+
+class Pooling(runner: () -> Unit) {
```

Comment:
This should be `Polling`, not `Pooling`, correct?

Reply:
Yes! Done!